BambooTracker/gui/configuration_dialog.cpp

```diff
@@ -24,57 +25,98 @@ ConfigurationDialog::ConfigurationDialog(std::weak_ptr<Configuration> config, QW
 		emit applyPressed();
 	});
 
+	std::shared_ptr<Configuration> configLocked = config.lock();
```

Comment:
I mass-replaced all the separate `config.lock()`s and implicit unlocks with a single locked `configLocked`. Let me know if you'd rather not have that, if I should move it into a separate commit/PR, …

Reply:
It's OK to push the changes to this PR.